compiler/common-artifacts/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@
 find_package(PythonInterp 3.8 QUIET)
 find_package(PythonLibs 3.8 QUIET)
 
+if(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.12")
+  find_package(Python3 COMPONENTS Interpreter Development QUIET)
+  set(PYTHON_EXECUTABLE Python3_EXECUTABLE)
+  set(PYTHON_VERSION_MINOR Python3_VERSION_MINOR)
+endif()
```

Comment:
I'm not sure adding patche with version check is appropriate.
What I think, but also not confident so we can discuss in another issue about this topic.
- raise an issue to support 22.04 as this year is 2022.
- upgrade default cmake version to meet this
- and so on...

Reply:
OK, I will create an issue about this topic.
I think it's a bit early to upgrade default cmake version. I will upload it in a direction supports ubuntu 22.04 as well.

Reply:
I created that issue at #9432